eos-tech-support/eos-write-image

```diff
@@ -52,14 +53,14 @@ while true; do
 done
 
 command -v dd >/dev/null || {
```

Comment:
@wjt Please change these to `if`/`then`-style (`||` is really hard to read and maintain.

I think in other code we use something like `if [ "$(which dd)" == "" ]; then` for these command checking constructs.